From: Markus Koschany <apo@gambaru.de>
Date: Sat, 22 Nov 2014 23:54:59 +0100
Subject: typedef

Define GLsizeiptr and GLintptr explicitly to prevent a FTBFS.
This patch may be removed in the future when
https://bugs.debian.org/765933 gets fixed.

Bug: https://bugs.debian.org/769301
Forwarded: no
---
 j3d-core/src/native/ogl/gldefs.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/j3d-core/src/native/ogl/gldefs.h b/j3d-core/src/native/ogl/gldefs.h
index bf4434f..d20de17 100644
--- a/j3d-core/src/native/ogl/gldefs.h
+++ b/j3d-core/src/native/ogl/gldefs.h
@@ -65,6 +65,8 @@
 #include <X11/Xlib.h>
 #include <X11/Xutil.h>
 
+typedef ptrdiff_t GLsizeiptr;
+typedef ptrdiff_t GLintptr;
 #include <GL/gl.h>
 #include <GL/glx.h>
 #ifdef Java3D_undef__glext_h_
